src/content/en/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -0,0 +1,60 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Say goodbye to shadow-piercing CSS selectors.
+
+{# wf_updated_on: 2017-10-24 #}
+{# wf_published_on: 2017-10-24 #}
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: Say goodbye to shadow-piercing CSS selectors. #}
+{# wf_blink_components: Blink>DOM #}
+
+# Removing ::shadow and /deep/ in Chrome 63 {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Starting in Chrome 63, the shadow-piercing selectors `::shadow` and `/deep/`
```

Comment:
Maybe reword a bit... 'Starting in Chrome 63, you cannot use the shadow-piercing selectors .... to style content inside of a shadow root'.